javascript/javascript1/week4/week04homework.js

```diff
@@ -0,0 +1,70 @@
+let listToDo =[];
+console.log(listToDo);
+function getReply(command){
+    if(command == 'What is my name'){
+        console.log('name is not mentioned');
+     }
+    else if(command == 'Add fishing to my todo'){
+        let actString = command.substr(4,7);
+        listToDo.push(`${actString} added to your todo`);
+        console.log(`${actString} added to your todo`);
+    }
+    else if(command == 'Add singing in the shower to my todo'){
+        let singString = command.substr(4,25);
+        listToDo.push(`Add ${singString}`);
+    }
+    else if(command == 'Remove fishing from my todo'){
+        console.log(`"Removed fishing from your todo."`);
+    }
+    else if(command == 'What is on my todo?'){
+        console.log(listToDo);
+    }
+    else if(command == 'What day is it today?'){
+        let monthOfYear = ['january','february','march','april','may','june','july','august','september','november','december']
+        let month = new Date().getMonth();
+        month = monthOfYear[month];
+        let date = new Date().getDate()+'-'+ month +'-'+new Date().getFullYear();
+        console.log(date);
+    }
+    else if(command == 'what is 3 + 3'){
+        let value1 = command.substr(-5,1);
+        let value2 = command.substr(-1,1);
+        let operator =command.substr(-3,1);
+        let v1 = Number(value1);
+        let v2 = Number(value2); 
+        let Value = v1 + v2;
+        console.log(Value);
+    }
+    else if(command == 'what is 4 * 12'){
+        let value1 = command.substr(-5,1);
+        let value2 = command.substr(-1,2);
+        let operator =command.substr(-3,1);
+        let v1 = Number(value1);
+        let v2 = Number(value2); 
+        let Value = v1 * v2;
+        console.log(Value);
+    }
+    else if(command == 'Set a timer for 4 minutes'){
+        let time = command.substr(-9,1);
+        let timeToMiliseconds = time*60*1000;
+        console.log(`"Timer set for ${time} minutes"`);
+        setTimeout(function(){console.log('Timer done')},timeToMiliseconds);
+    }
+    else if(typeof command == 'string'){ 
```

Comment:
I do not understand this condition , as every command that is passed is a string.

Reply:
Yes I understand but also I have managed to place it in the order of the conditions. But I have made the changes in the code as you expected.

Reply:
Please update me once I can commit the PR , after reviewing the changes asap.

Reply:
I don't want you to write the code as I expected, but instead to understand, the condition you gave there, which checks if the command is a string.
All the commands that got in to this function is a string, so that does not make sense to have a general condition to check the name command.

Reply:
commit is what you do to the PR, I think you are talking about approve the PR to merge.